bin/aws-s3-creator.ts

```diff
@@ -25,7 +25,7 @@ const deployEnvironment = process.env.ENVIRONMENT!;
 
 const appName = process.env.APP_NAME!;
 const owner = process.env.OWNER!;
```

Comment:
**Suggestion:** Add a check to ensure `process.env.APP_NAME` and `process.env.OWNER` are not empty before using them to avoid potential runtime errors.
> 🎇  **High Priority** 
 > [category: `possible issue`, importance: `**9**`]
> **Reason:** This suggestion is crucial as it ensures that essential environment variables `APP_NAME` and `OWNER` are checked for non-emptiness before use, preventing potential runtime errors from undefined values.
 

```suggestion
const appName = process.env.APP_NAME ? process.env.APP_NAME : '';
const owner = process.env.OWNER ? process.env.OWNER : '';
```